odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/repository/reactive/ReactiveSearchEntrypointRepository.java

```diff
@@ -37,7 +37,7 @@ public interface ReactiveSearchEntrypointRepository {
 
     Mono<Integer> updateChangedOwnershipVectors(final long ownershipId);
 
-    Mono<Integer> updateChangedLabelVector(final long labelId);
+    Mono<Integer> updateChangedTagVector(final long tagId);
```

Comment:
you already have 
`Mono<Integer> updateChangedTagVectors(final long tagId);`
method above. 
Need to integrate logic into it